src/pages/index.jsx

```diff
@@ -14,22 +14,18 @@ const HomePage = () => (
     <SEO title="Home" />
     <Container fixed>
       <section>
-        <Grid container spacing={2}>
```

Comment:
Is there a reason we aren't using Grid components? I personally prefer it so that all website elements are responsive to the same breakpoints

Reply:
I removed it from the Hero because I don't know of a way to flip the order with the Grid component. I don't think it looks good for it to have the multi logo, large multi text again, and then the graphic. This way we can flip the order with the flex direction.

Reply:
Ahh gotcha! I'll probably look into it because I'm genuinely curious, but that's definitely a later thing. As long as we're using the same Material UI breakpoints, then we're good